Makefile

```
@@ -86,12 +86,21 @@ $(AMD64_BINS): GOARCH=amd64
 $(ARM64_BINS): GOARCH=arm64
 
 .PHONY: build
-build: $(DRIVER_BINS)
+build: vet fmt $(DRIVER_BINS)
 
-$(DRIVER_BINS): vet fmt
+$(DRIVER_BINS):
 	@CGO_ENABLED=0 GOTOOLCHAIN=local GOOS=linux GOARCH=$(GOARCH) \
 		go build -ldflags "${LDFLAGS}" -o $@ ./cmd
 
+# For CI use. Runs `make build` inside Docker
+# container using an official golang image.
+.PHONY: docker-build
+docker-build:
+	docker run --rm \
+		--workdir /go/src/github.com/racktopsystems/brickstor-csi-driver \
+		--volume ./:/go/src/github.com/racktopsystems/brickstor-csi-driver \
```

I think we should bias towards using `$(CURDIR)` here.
```suggestion
		--volume $(CURDIR)/:/go/src/github.com/racktopsystems/brickstor-csi-driver \
```